lattices/LatticeMath/LatticeStatistics.tcc

```diff
@@ -924,7 +921,7 @@ Bool LatticeStatistics<T>::generateStorageLattice() {
                 outLatt, *pInLattice_p,
                 collapser, IPosition(cursorAxes_p),
                 newOutAxis,
-                pProgressMeter.null() ? NULL : &*pProgressMeter
+                !pProgressMeter ? NULL : pProgressMeter.get()
```

Comment:
I find negative conditionals to be more difficult to read than positive, so I'd rewrite
`
pProgressMeter ? pProgressMeter.get() : NULL
`

Reply:
done